WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -750,7 +750,7 @@ def setJobStatus(self, jobID, status='', minor='', application=''):
       return ret
     application = ret['Value']
 
-    cmd = 'REPLACE JobsStatus (JobID,Status,MinorStatus,ApplicationStatus) VALUES (%d,%s,%s,%s)' % (
+    cmd = 'REPLACE INTO JobsStatus (JobID,Status,MinorStatus,ApplicationStatus) VALUES (%d,%s,%s,%s)' % (
```

Comment:
For completeness: `REPLACE INTO` is not SQL standard. standard would be `insert.. on duplicate key update`

Reply:
I know... but is it a problem? Anyway we are on MySQL only.

Reply:
no no, I am just saying, since we regularly get the question..